src/app/screens/LNURLWithdraw/index.tsx

```diff
@@ -67,7 +69,12 @@ function LNURLWithdraw() {
       });
 
       if (response.data.status.toUpperCase() === "OK") {
-        setSuccessMessage(t("success"));
+        setSuccessMessage(
+          t("success", {
+            amount: `${valueSat} SATS${showFiat ? ` (${fiatValue})` : ``}`,
```

Comment:
Should we add the "space" after the `SATS`? This aligns it with the other usecases.